test/source/dextool_test/utils.d

```diff
@@ -575,3 +576,218 @@ void dirContentCopy(string src, string dst) {
             setAttributes(dst_f, attrs | S_IXUSR);
     }
 }
+
+// Check if commands are logged or not in the logfile (controlled with 'expected'-parameter)
+auto checkCommandsInLogFile(string[] commands, string file, bool expected) {
```

Comment:
This can be replaced by the utilities moved from mutate?

Reply:
Yes, the reason I left the function was that it specified what you were looking for (commands in a logfile). But fair enough, it might be unnecessary. I'll remove it.